library/spdm_common_lib/libspdm_com_opaque_data.c

```diff
@@ -58,17 +58,49 @@ size_t libspdm_get_opaque_data_supported_version_data_size(
         size = sizeof(spdm_general_opaque_data_table_header_t) +
                sizeof(secured_message_opaque_element_table_header_t) +
                sizeof(secured_message_opaque_element_supported_version_t) +
-               sizeof(spdm_version_number_t);
+               sizeof(spdm_version_number_t) *
+               spdm_context->local_context.secured_message_version.spdm_version_count;
     } else {
         size = sizeof(secured_message_general_opaque_data_table_header_t) +
                sizeof(secured_message_opaque_element_table_header_t) +
                sizeof(secured_message_opaque_element_supported_version_t) +
-               sizeof(spdm_version_number_t);
+               sizeof(spdm_version_number_t) *
+               spdm_context->local_context.secured_message_version.spdm_version_count;
     }
     /* Add Padding*/
     return (size + 3) & ~3;
 }
 
+/**
+ * Return the size in bytes of opaque data supproted version.
```

Comment:
"supproted" -> "supported"

Reply:
There are 4 "supproted" typo in total in this file. And they are all fixed in the latest commit, please check. Thank you.